src/compressed_tensors/compressors/model_compressors/model_compressor.py

```diff
@@ -182,7 +182,22 @@ def from_pretrained_model(
             algorithm
         :return: compressor for the configs, or None if model is not compressed
         """
-        # reconstruct config from schemes attached to modules
+
+        compression_formats = None
+        if quantization_format is not None:
+            # llmcompressor incorrectly passes in a CompressionFormat when
+            # the value string is expected - handle both cases
+            if isinstance(quantization_format, (str, CompressionFormat)):
+                quantization_format = [quantization_format]
+
+            compression_formats = quantization_format
```

Comment:
FYI yhis parsing logic is duplicated in `from_pretrained_model` and `decompress_model`.